common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -62,6 +66,24 @@ static mon_wlan_hal::Event dwpal_to_bwl_event(const std::string &opcode)
     return mon_wlan_hal::Event::Invalid;
 }
 
+static mon_wlan_hal::Event dwpal_nl_to_bwl_event(uint8_t cmd, bool waiting_for_results_ready)
+{
+    switch (cmd) {
+    case NL80211_CMD_TRIGGER_SCAN:
+        return mon_wlan_hal::Event::Channel_Scan_Triggered;
```

Comment:
Here you mixed camel case and `_` - please don't do that. Change it either to camel case `ChannelScanTriggered` or all lower case `channel_scan_triggered` (nitpick)

Reply:
#665